src/main/kotlin/io/github/mojira/arisa/domain/Version.kt

```diff
@@ -1,9 +1,12 @@
 package io.github.mojira.arisa.domain
 
 import arrow.core.Either
+import java.time.Instant
 
 data class Version(
+    val name: String,
```

Comment:
This needs to be kept as ID like before, otherwise the addVersionByID from jira/Calls.kt will not work